pod/perlvar.pod

```diff
@@ -1840,7 +1840,7 @@ Current state of the interpreter.
     $^S         State
     ---------   -------------------------------------
     undef       Parsing module, eval, or main program
-    true (1)    Executing an eval
+    true (1)    Executing an eval or try block
```

Comment:
This makes it sound like it's only `1` when executing an `eval` bloc, not a string `eval`. Maybe something like

```
undef     Parsing module, eval string, or main program
true (1)  Executing try block or eval string or block.
```

Reply:
Or leave out "block": "Executing an eval or try"